download-references.nf

```diff
@@ -135,9 +135,9 @@ process download_star_fusion {
 
     script:
     """
-    aws s3 --no-sign-request --region eu-west-1 sync s3://ngi-igenomes/igenomes/Homo_sapiens/NCBI/GRCh38/Annotation/CTAT/ .
-    tar -xf GRCh38_gencode_v32_CTAT_lib_Dec062019.tar.xz --strip-components 5
-    rm GRCh38_gencode_v32_CTAT_lib_Dec062019.tar.xz
+    aws s3 --no-sign-request --region eu-west-1 sync s3://ngi-igenomes/Homo_sapiens/Ensembl/GRCh38/Genome/CTAT/ctat_star_fusion_1_8_1.tar.gz .
```

Comment:
`aws s3 --no-sign-request --region ${params.awsregion} sync s3://ngi-igenomes/Homo_sapiens/Ensembl/GRCh38/Genome/CTAT/ctat_star_fusion_1_8_1.tar.gz .`

Reply:
just wondering if this is actually a good idea, as `awsregion` is not set up to begin with.
Maybe we should add a default value here just in case